### DISCUSSION TITLE
Add anchor positioning

### DISCUSSION THREAD

**Discussion by seb-jean** [tailwindlabs/tailwindcss#13790] [Ideas]
Hello,

It would be interesting to add anchor positioning for version 4 of Tailwind CSS.

**Comment by rakurtz** [comment]
Is there no discussion about this anywhere?

**Comment by davidvmckay** [comment]
this works reasonable well: https://github.com/toolwind/anchors

**Reply by brandonmcconnell** [reply]
@seb-jean @rakurtz Check it out!

The plugin supports both Tailwind CSS v3 and v4, providing simple, declarative APIs for working with anchors and all properties related to the [CSS Anchor Positioning](https://www.w3.org/TR/css-anchor-position-1/) specification.

**Reply by rakurtz** [reply]
looks nice! thanks! will check it out

**Comment by notnotjake** [comment]
I would really like this. CSS anchors are really powerful and useful. And it is not something that we can create a util for.

**Reply by brandonmcconnell** [reply]
The [toolwind/anchors](https://github.com/toolwind/anchors) plugin (crafted by yours truly) works incredibly well. I've used this myself on several projects, and it plays nicely with the [oddbird/css-anchor-positioning](https://github.com/oddbird/css-anchor-positioning) polyfill too.

**Reply by sirkostya009** [reply]
i'd love tailwindcss to have this feature built-in
